server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -91,6 +92,9 @@ public Invocations getInvocations() {
 
             // Stats
             .invocation().methods(GET).path("/v2/cache-managers/{name}/stats").handleWith(this::getStats)
+
+            // Caches
+            .invocation().methods(GET).path("/v2/cache-managers/{name}/caches").handleWith(this::getCaches)
```

Comment:
Were already have something similar in the ```CacheResourceV2```: https://github.com/infinispan/infinispan/blob/7ad546e0dfbe41a8155e83cf1bf6c4f811ab57f7/server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java#L69

I think you should move your logic there, maybe with a request parameter ```?detailed=true```

Reply:
Hmm...ok. But we need to unify those two...One returns only the names, this one the name + the details. What about bring the other one here?

Reply:
but the other are all the caches... well now that we have a single cache manager both should bring the same amount of caches...

Reply:
I agree we should decide how to do this 

@tristantarrant are you using this method in the CLI ? the caches one.

Reply:
Until we know, let's keep it here, and mark the API as 'experimental' in the docs :)

Reply:
you mean this new api method ?

Reply:
yep I did not change the docs